tests/integration/release/test_runner.py

```diff
@@ -1,9 +1,12 @@
 import os
```

Comment:
All the changes in this file will eventually be replaced by jobmon logic and this is just here to check that the other changes are working.